src/GenAPI/Microsoft.DotNet.GenAPI/Filters/DelegateMembersFilter.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions;
+
+namespace Microsoft.DotNet.GenAPI
+{
+    internal class DelegateMembersFilter : ISymbolFilter
+    {
+        /// <inheritdoc />
+        public bool Include(ISymbol member)
```

Comment:
Same as in the ImplicitSymbolsFilter, please add a tiple slash comment here instead of inheritdoc to explain what this filter does.